.github/workflows/release.yaml

```diff
@@ -36,4 +36,4 @@ jobs:
         with:
           charts_dir: helm
         env:
-          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          CR_TOKEN: "${{ secrets.CR_TOKEN }}" # using a personal access token here so that the build-on-relase workflow can be triggered
```

Comment:
were we missing permissions on GITHUB_TOKEN? https://docs.github.com/en/actions/reference/authentication-in-a-workflow#about-the-github_token-secret

Reply:
No, actions triggered by the github bot cannot trigger workflows, to avoid infinite loops: https://docs.github.com/en/actions/reference/events-that-trigger-workflows
>Note: You cannot trigger new workflow runs using the GITHUB_TOKEN. For more information, see "Triggering new workflows using a personal access token."